src/views/AddPropertyManager/index.jsx

```diff
@@ -39,57 +37,55 @@ const AddPropertyManager = () => {
   const [propertyOptions, setPropertyOptions] = useState([]);
   const [propertySearchResults, setPropertySearchResults] = useState([]);
   const [showAddProperty, setShowAddProperty] = useState(false);
-
-  useMountEffect(() => getProperties());
```

Comment:
The move to getting properties "onBlur - but only once" doesn't seem to work and I'm not sure it's necessary to remove the call from the loading of the page. It's asynchronous so the loading of the page doesn't wait for the property call to complete, but if we put it in the onBlur of the formik component then the user has to wait after they clicked on the search field for the results to come back. We also want the list to appear ASAP when they click even without searching in the field.
I think we can switch it back to querying the properties on page load and take the "propertyRefresh" state out as well.

Reply:
> I think we can switch it back to querying the properties on page load and take the "propertyRefresh" state out as well.

ok